style-guide/check-howto.py

```diff
@@ -0,0 +1,17 @@
+# -*- coding: utf-8 -*-
+"""How to construct."""
```

Comment:
Is it possible to only check for this in headlines and titles?

Reply:
Yes, it is! Fixed this!